docs/agents/quickstart.md

```diff
@@ -0,0 +1,171 @@
+# Quickstart a Python Agent
+
+Welcome! Agents are programs that can act autonomously to achieve a predefined goal understand, while
+understanding and reacting to their environment.
+
+NearAI agents can communicate with each other, and have access to a wide range of capabilities, including
+authentication, tools, apis, smart contract calls, and more.
+
+Let's build our first agent on Near AI!
+
+---
+
+## Pre-Requisites
+
+To get started, you will need to have a [Near account](https://wallet.near.org/), and install the [Near AI CLI](https://github.com/nearai/nearai/#setup):
```

Comment:
You may want to point them directly to a wallet that works well with NearAi, like Bitte. As some of the other ones do not support the required 'SignMessage' functionality

Reply:
ah, nevermind :)  you mention the same suggestion below